Lab 1 "External (file) sorts"/Lab 1 "External (file) sorts"/main.cpp

```diff
@@ -0,0 +1,60 @@
+//
+//  main.cpp
+//  Lab 1 "External (file) sorts"
+//
+//  Created by Antony Miroshnichenko on 20.02.2024.
+//
+
+
+
+#include "Funcs.hpp"
+
+enum SortTypes {Прямая_Сортировка = 0, Естественная_Сортировка = 1};
```

Comment:
Ну кириллицу-то зачем сюда?...
```suggestion
enum SortType
{
    DirectMerge,
    NaturalMerge
};
```
Как и в случае с классами, множественное число использовать не следует. Оно +- нормально выглядит при объявлении класса, но будет путать при создании переменных:
```c++
    SortTypes sortType = SortTypes::DirectMerge;
```

P.S. Т.к. enum используется в функции, то и объявлен он должен быть там же.